torchvision/models/segmentation/deeplabv3.py

```diff
@@ -47,21 +47,29 @@ def __init__(self, in_channels, out_channels, dilation):
         super(ASPPConv, self).__init__(*modules)
 
 
-class ASPPPooling(nn.Sequential):
+class ASPPPooling(nn.Module):
+    __constants__ = ['0', '1', '2', '3']
+
     def __init__(self, in_channels, out_channels):
-        super(ASPPPooling, self).__init__(
-            nn.AdaptiveAvgPool2d(1),
-            nn.Conv2d(in_channels, out_channels, 1, bias=False),
-            nn.BatchNorm2d(out_channels),
-            nn.ReLU())
+        super(ASPPPooling, self).__init__()
+        self.add_module('0', nn.AdaptiveAvgPool2d(1))
+        self.add_module('1', nn.Conv2d(in_channels, out_channels, 1, bias=False))
+        self.add_module('2', nn.BatchNorm2d(out_channels))
+        self.add_module('3', nn.ReLU())
 
     def forward(self, x):
         size = x.shape[-2:]
-        x = super(ASPPPooling, self).forward(x)
+        # workaround while torchscript doesn't support overriding forward in nn.Sequential
+        x = getattr(self, '0')(x)
```

Comment:
does `self.0` not work here instead ? TorchScript doesn't support `getattr`

Reply:
`self.0` is an invalid syntax in python unfortunately.

This is weird, because I tried compiling this code and running it, and it seemed to work fine and give the same result as the non-compiled version

Reply:
This looks pretty hacky :-)